10/31　~16:00
作業したことをメモる。

①DC:A6:32:8C:8F:AA　と　②E4:5F:01:25:11:B6
やりたいこと：Send and Reserveのテスト　まずは両ラズパイをBLE通信できる状態にするところから始める

https://qiita.com/shippokun/items/0953160607833077163f　参考

試してみるが上手くいかない　　→　　最初からなぞろうの会　
①②に関連パッケージインストール＋bluezインストール なおできない。　お互いが認識できてない。
https://pc.atsuhiro-me.net/entry/2021/06/03/155123　
boot/config.txt　をいじればどうにかなる？　次回そこから。

11/8 
boot/config.txt に　# dtoverlay=disble-bt を追加。
再度BLE通信挑戦。　成功！！

main.py 行けるか試してみる？
書いてあること試す
$ sudo apt-get install git
$ git clone https://github.com/T3pp31/BLE_Blockchain
$ cd BLE_Blockchain
$ python3 install_package.py
setting.jsonをお互いのラズパイで書き換え。

送信されてくるデータを待っているが動いているのか不明。　実行は出来てるっぽい。　l2cap_server.pyの送信されてくるデータを待っています。までは見れる。
#32について　言われなくてもそれはわかってる。どうすればいのかはわからん。

11/9
main.py いじりながらやっていくしか。 ②にVScodeいれた。
11/10 
一旦形式は出来た。動かしながらやってみる。
11/11
main.py の送信ブロックチェーンを削った部分を試してみる。
実行結果：changing discoverable on succeeded 改行して　学籍番号 near_device 備考　0、1は出たけどその後Traceback。
49行目のdelete_excess_data(df)に対してTypeError: 'module' object is not callable　一旦ここまで。
main.pyで試すのはきつい気がする。　独自で動くかチェック。宮前君と連携とりたい。

11/15
試してみた？とりあえずgithubに作ったの載せた。①をssh接続できるようにした。https://qiita.com/c60evaporator/items/2384416f1122ae124f50　参照
ポートかなんかで引っ掛かる。明日やってみて考える？
